apps/playground-web/src/components/account-abstraction/5792-get-capabilities.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+import { sepolia } from "thirdweb/chains";
+import {} from "thirdweb/extensions/erc1155";
+import {
+  ConnectButton,
+  useActiveWallet,
+  useCapabilities,
+  useWalletInfo,
+} from "thirdweb/react";
+import { createWallet } from "thirdweb/wallets";
+import { THIRDWEB_CLIENT } from "../../lib/client";
+import CodeClient from "../code/code.client";
+
+export function Eip5792GetCapabilitiesPreview() {
```

Comment:
This would be a really cool utility to have on the dashboard too. Connect your wallet, see all the capabilities it has in plaintext